plugins/outputs/elasticsearch/elasticsearch.go

```diff
@@ -424,7 +424,7 @@ func (a *Elasticsearch) createNewTemplate(templatePattern string) (*bytes.Buffer
 	return &tmpl, nil
 }
 
-func (a *Elasticsearch) GetTagKeys(indexName string) (string, []string) {
+func GetTagKeys(indexName string) (string, []string) {
```

Comment:
Do we need to export this function?

Reply:
@srebhan Yes, you’re right, they can be unexported. Do you mind if I take care of this as part of fixing linter findings for `revive:exported` effort?

Reply:
No fine with me.